tests/features/sbom-explorer.feature

```diff
@@ -0,0 +1,178 @@
+    Then list of SBOM Packages should be limited only to matching one
+    And list of SBOM Packages should be sorted alphabetically by Package name
+    When replacing Filter field input with value not matching any SBOM Package name
+    Then list of SBOM Packages should be empty # showing info about no-match-found?
+    When clearing the Filter field input
+    Then list of SBOM Packages should show original list of SBOM Packages again # not "all", still paginated
+
+    When looking at each individual Package entry
+    Then SBOM Package name should be visible
+    And SBOM Package version should be visible
+    And SBOM Package qualifiers should be visible
+
+  Scenario: View paginated list of SBOM Packages
+    Given There is ingested SBOM with more packages than fits in a page
+    When User visits SBOM details Page
+    And User selects Packages tab
+    Then Pagination of results works
+
+  Scenario: View expanded SBOM Package with sufficient data
+    # TODO: clarify 'sufficient data' (in all parts of this scenario)
+    Given There is ingested SBOM with Packages with sufficient data
+    When user visits SBOM details page
+    And user selects Packages tab
+    And user finds package with enough details
+    Then package Package entry should be expandable
+
+    When expanding Package entry
+    Then SBOM Package details should be visible
+
+    When looking at SBOM Package details
+    Then Packages column should be visible
+    Then Details column should be visible
+    Then Qualifiers column should be visible
+    Then Version column should be visible
+
+    When looking at each entry in SBOM Package details
+    Then there is link in Packages column
+    And link in Packages column leads to Package Explorer page
+
+
+  Scenario: View all SBOM Vulnerabilities
+    Given there is ingested SBOM which is affected by Vulnerabilities
+    When user visits SBOM details page
+    And user selects Vulnerabilities tab
+    And looking at the tab content
+    Then Vulnerability Risk Profile circle should be visible
+    And Vulnerability Risk Profile should show summary of total amounts of vulnerabilities
+    And SBOM Name should be visible #
+    And SBOM Version should be visible
+    And SBOM Creation date should be visible
+    And list of related Vulnerabilities should be visible
+    And list of related Vulnerabilities should be paginated
+
+    When user looks at list of related Vulnerabilities
+    Then list should be sorted by CVSS descending # e.g. from Critical first
+    And all entries should have expected columns
+    # (CVE) ID, Description, CVSS, Count of Affected dependencies, Published date, Updated date
+    # Note: if to expand all the columns to individual Then's may depend on implementation
+    #       how we can 'loop' over all entries (seems should not be done in cucumber test description?)
+    When user selects any column, except description, to sort by
```

Comment:
Like for pagination, we should have generic scenario for column header sorting as well.

Reply:
Yep, not sure if scenario though, as I'm not sure we can include/invoke feature files/scenarios ... maybe it should be just universal step implementation. Scenario would map it to corresponding data-situation and table/list.

Reply:
Similar to pagination, that depends on actual implementation and as such is out of scope of this PR.